### PR TITLE
Added capability to remove user from Host Team

### PR DESCRIPTION
**Pull request by anweshknayak**
@deshraj @spyshiv  @aka-jain  please review

### CODE REVIEW

**Comment by deshraj** [comment]
@anweshknayak: Just an advice, Please try to make meaningful commit messages. It helps everyone to keep track of what work was done in that particular commit. :-)

**Comment by anweshknayak** [comment]
@deshraj I will make sure of that from next time :)

**Comment by deshraj** [comment]
@anweshknayak can you please remove the part of #477 since @Ram81 was already working on that issue. Please take of these things from next time otherwise it leads to duplicate effort. :-) 
There are other plenty of issues to solve.

**Comment by anweshknayak** [comment]
@deshraj my bad . I didn't notice.

**Comment by anweshknayak** [comment]
@deshraj @spyshiv please review

**Comment by deshraj** [comment]
@anweshknayak can you please resolve conflicts?

**Comment by anweshknayak** [comment]
@deshraj sure

**Comment by anweshknayak** [comment]
closing this since already fixed by @aka-jain in #574

File: frontend/src/js/controllers/challengeHostTeamsCtrl.js [hunk 1]
```diff
@@ -7,9 +7,10 @@
         .module('evalai')
         .controller('ChallengeHostTeamsCtrl', ChallengeHostTeamsCtrl);
 
-    ChallengeHostTeamsCtrl.$inject = ['utilities', '$state', '$http', '$rootScope'];
+    ChallengeHostTeamsCtrl.$inject = ['utilities', '$scope' , '$state', '$http',
```
Comment (aka-jain) [inline frontend/src/js/controllers/challengeHostTeamsCtrl.js]: @anweshknayak Please don't use scope.

File: frontend/src/views/web/challenge-host-teams.html [hunk 2]
```diff
@@ -16,8 +16,9 @@
                 <div class="ev-md-container ev-card-body exist-team-card">
                     <ul>
                         <li ng-repeat="(key, value) in challengeHostTeams.existTeam.results">
-                            <i class="fa fa-users" aria-hidden="true"></i>
-                            <div class="show-member-title pointer"><strong class="text-med-black">Team: </strong>{{value.team_name}}<i class="fa fa-plus list-icon text-highlight w-300 right"></i>
+                             <i class="fa fa-users" aria-hidden="true"></i>
+                              <div class="show-member-title pointer"><strong class="text-med-black">Team: </strong>{{value.team_name}}
```
Comment (aka-jain) [inline frontend/src/views/web/challenge-host-teams.html]: Looks like that it isn't indented properly.

**Comment by aka-jain** [comment]
Also solve conflicts too.

**Comment by anweshknayak** [comment]
@aka-jain can you please check this PR (https://github.com/Cloud-CV/EvalAI/pull/574/files) Isn't it the same issue ?

**Comment by aka-jain** [comment]
Yep, I have done some part of code before. But there was some error from backened side. And that was the issue. I was going to tell you about this. Sorry for late information.
I'm closing this PR as it is of no use. 

@deshraj Will you please check the delete API for host team?

**Pull request closed without merging**